ami_assets/snippets/iframe-styling.html

```diff
@@ -0,0 +1,7 @@
+<style id="iframe-styling">
+    iframe {
+        /*margin: 0;*/
```

Comment:
Matching latest code on AMI.

```suggestion[
        min-height: 160vh;
```